Don't override MacPorts CPPFLAGS, CXXFLAGS, or LDFLAGS. Don't disable
deprecated GTK functions because that causes the build to fail.
--- Makefile.orig	2004-02-09 11:08:41.000000000 -0600
+++ Makefile	2022-03-31 05:23:35.000000000 -0500
@@ -2,12 +2,10 @@
 
 CFLAGS := $(if $(RPM_OPT_FLAGS), $(RPM_OPT_FLAGS), $(CFLAGS))
 
-LDFLAGS = $(shell pkg-config --libs gtk+-2.0)
+LDFLAGS += $(shell pkg-config --libs gtk+-2.0)
 CFLAGS += -Wall
-CFLAGS += $(shell pkg-config --cflags gtk+-2.0) -DGTK_DISABLE_BROKEN -DGTK_DISABLE_DEPRECATED
+CFLAGS += $(shell pkg-config --cflags gtk+-2.0) -DGTK_DISABLE_BROKEN
 CFLAGS += -DPROJNAME='"$(PROJNAME)"' -DVERSION='"$(VERSION)"'
-CPPFLAGS =
-CXXFLAGS =
 
 SRC = util stock theme_sel font_sel preview_pane about_dialog mainwin main
 
